eclair-core/src/test/scala/fr/acinq/eclair/integration/MessageIntegrationSpec.scala

```diff
@@ -94,6 +102,7 @@ class MessageIntegrationSpec extends IntegrationSpec {
     val probe = TestProbe()
     val eventListener = TestProbe()
     nodes("A").system.eventStream.subscribe(eventListener.ref, classOf[OnionMessages.ReceiveMessage])
+    sleep(100 millis)
     bob.sendOnionMessage(Nil, Left(nodes("A").nodeParams.nodeId), Some(Nil), hex"3f00").pipeTo(probe.ref)
```

Comment:
There should be a check that `probe` did receive a  `SendOnionMessageResponse` here ?